.github/scripts/sync_labels.py

```diff
@@ -48,7 +48,7 @@ def get_linked_pr_from_issue_number(repo, number):
 
 def get_linked_issues_based_on_pr_body(repo, number):
     pr = repo.get_pull(number)
-    pattern = fr'Fixes:? (?:#|{repo}#|https://github.com/{repo}/issues/)(\d+)'
+    pattern = r'Fixes:? (?:#|{repo}#|https://github.com/{repo}/issues/)(\d+)'
```

Comment:
I don't understand what was the problem and how this change fixes it.
Please describe what this patch is doing in the git commit log.

Specifically, I don't see how the pattern cannot be formatted. Shouldn't `{repo}` be expanded?